pkg/apis/gcp/compute/v1alpha1/types_test.go

```diff
@@ -90,3 +91,103 @@ func TestGKECluster(t *testing.T) {
 	g.Expect(c.Delete(ctx, fetched)).NotTo(HaveOccurred())
 	g.Expect(c.Get(ctx, key, fetched)).To(HaveOccurred())
 }
+
+func TestParseClusterSpec(t *testing.T) {
+	tests := []struct {
+		name string
+		args map[string]string
+		want *GKEClusterSpec
+	}{
+		{
+			name: "nil properties",
+			args: nil,
+			want: &GKEClusterSpec{ReclaimPolicy: DefaultReclaimPolicy},
+		},
+		{
+			name: "empty properties",
+			args: nil,
```

Comment:
should this be an empty map instead of a nil pointer to match test intent implied by its name?

Reply:
Good catch, fixed.